cdp_scrapers/instances/portland.py

```diff
@@ -318,18 +424,27 @@ def get_votes(self, minute_section: Tag) -> Optional[List[Vote]]:
         vote_list = []
         for vote_element in vote_elements:
             vote = vote_element.text.strip()
+            # at this point vote string is like
+            # Commissioner Jo Ann Hardesty Absent
+            # Commissioner Mingus Mapps Yea
+            is_absent = "Absent" in vote
+            vote = vote.replace(" Absent", "")
```

Comment:
When we're doing raw string searches it could be worth normalizing the data first by doing something like `.lower()` on the string. Might not be necessary in this case but something to consider

Reply:
Totally agree. Better to be safe than sorry.